zuul-discovery/build.gradle

```diff
@@ -5,7 +5,7 @@ dependencies {
     implementation libraries.guava
     implementation libraries.slf4j
 
-    implementation "com.netflix.ribbon:ribbon-loadbalancer:${versions_ribbon}"
```

Comment:
had to change this because I added the constructor that takes a DynamicServerListLoadBalancer